src/shelly_switch.hpp

```diff
@@ -74,6 +74,14 @@ class ShellySwitch : public Component, public mgos::hap::Service {
   bool dirty_ = false;
 
   ShellySwitch(const ShellySwitch &other) = delete;
+
+  void AddPowerMeter(uint16_t *iid);
+  void PowerMeterTimerCB();
+  mgos::Timer power_timer_;
+  mgos::hap::Characteristic *power_char_;
+  mgos::hap::Characteristic *total_power_char__;
```

Comment:
one underscore is sufficient.
also please initialize with nullptr

Reply:
oh ©️ 🍝